providers/ofrep/src/main/java/dev/openfeature/contrib/providers/ofrep/OfrepProviderOptions.java

```diff
@@ -0,0 +1,32 @@
+package dev.openfeature.contrib.providers.ofrep;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import java.net.ProxySelector;
+import java.time.Duration;
+import java.util.concurrent.Executor;
+import lombok.Builder;
+import lombok.Getter;
+
+/**
+ * Options for configuring the OFREP provider.
+ */
+@Getter
+@Builder(builderClassName = "Builder", buildMethodName = "build")
+public class OfrepProviderOptions {
+
+    @Builder.Default
+    private final String baseUrl = "http://localhost:8016";
+
+    @Builder.Default
+    private final ProxySelector proxySelector = ProxySelector.getDefault();
+
+    @Builder.Default
+    private final Executor executor = Runnable::run;
```

Comment:
I think the default should be something like _Executors.newFixedThreadPool(DEFAULT_THREAD_POOL_SIZE)_ with default value 4-8 and configurable or _Executors.newCachedThreadPool()_, otherwise it can lead to too many threads theoretically.